Source/WebInspectorUI/UserInterface/Views/MultiSidebar.js

```diff
@@ -322,7 +364,48 @@ WI.MultiSidebar = class MultiSidebar extends WI.Sidebar
     {
         this.dispatchEventToListeners(WI.Sidebar.Event.WidthDidChange, {sidebar: event.target, newWidth: event.data.newWidth});
     }
+    
+    _recalculateHeight(newHeight)
+    {
+        console.assert(newHeight);
+        let maximumHeight = this.element.parentElement.offsetHeight - WI.TabBrowser.MinimumHeight;
+        
+        // Need to add 1 because of the 1px border-top.
+        newHeight = Math.ceil(Number.constrain(newHeight, this.minimumHeight + 1, maximumHeight));
+        this.element.style.height = `${newHeight}px`;
 
+        if (this.collapsed)
+            return;
+        
+        for (let sidebar in this.sidebars) {
+            sidebar.selectedSidebarPanel?.needsLayout(WI.View.LayoutReason.Resize);
+        }
+
+        this.dispatchEventToListeners(WI.MultiSidebar.Event.HeightDidChange, {newHeight});
+    }
+    
+    _adjustPosition()
+    {
+        const isNarrow = WI.layoutMode === WI.LayoutMode.Narrow;
+        this.element.classList.toggle("on-bottom", isNarrow);
+        
+        if (!isNarrow) {
+            this.element.style.height = "";
+            this._resizer?.element.remove();
```

Comment:
Where does `this._resizer` come from in this context? 
If it's the resizer for the height, perhaps making it explicit like `this._heightResizer` would help avoid confusion.

@dcrousso I'm not sure whether this should become a setter/getter on `MultiSidebar`. The fact that `this._resizer` comes into existence just in this handler seems odd.

Reply:
Done.